src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ReleasePrisonerService.kt

```diff
@@ -48,12 +54,25 @@ class ReleasePrisonerService(
   private fun OffenderBooking.isActiveIn(): Result<OffenderBooking> =
     if (!isActive) {
       failure(BadRequestException("Booking $bookingId is not active"))
-    } else if (!this.isIn) {
+    } else if (!isIn) {
       failure(BadRequestException("Booking $bookingId is not IN"))
     } else {
       success(this)
     }
 
+  private fun OffenderBooking.release(toLocation: AgencyLocation, releaseTime: LocalDateTime, reasonCode: String) =
+    apply {
+      inOutStatus = "OUT"
+      isActive = false
+      bookingStatus = "C"
+      livingUnitMv = null
+      assignedLivingUnit = null
+      location = toLocation
+      bookingEndDate = releaseTime
+      statusReason = "REL-$reasonCode"
+      commStatus = null
+    }
```

Comment:
I forgot to actually update the booking until I plumbed in the new service and the integration tests fell about laughing.